hibernate-reactive-core/src/test/java/org/hibernate/reactive/ReactiveSessionTest.java

```diff
@@ -1,17 +1,30 @@
 package org.hibernate.reactive;
 
-import io.vertx.ext.unit.TestContext;
-import org.hibernate.LockMode;
-import org.hibernate.cfg.Configuration;
-import org.junit.Test;
+import static org.junit.Assume.assumeTrue;
+
+import java.util.Objects;
+import java.util.concurrent.CompletionStage;
 
 import javax.persistence.Entity;
 import javax.persistence.Id;
 import javax.persistence.Table;
-import java.util.Objects;
-import java.util.concurrent.CompletionStage;
+
+import org.hibernate.LockMode;
+import org.hibernate.cfg.Configuration;
+import org.hibernate.reactive.containers.DatabaseConfiguration;
+import org.hibernate.reactive.containers.DatabaseConfiguration.DBType;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import io.vertx.ext.unit.TestContext;
 
 public class ReactiveSessionTest extends BaseReactiveTest {
+	
+	@BeforeClass
+	public static void beforeAll() {
+		// TODO: @AGG investigate this test class for DB2
```

Comment:
All of the tests in this class work with DB2 when I run them individually, but when I run the whole class it hangs and times out. I'm hoping this is just a test cleanup issue, perhaps with `cleanDB()` and looking at recent commits I think @gavinking may have already fixed it. I spent some time looking into it but couldn't quickly figure it out and decided to move on with this PR and fix it later. Since this PR touches almost every test class I wanted to do this quickly to avoid merge conflict hell.

Reply:
I've raised a separate issue to track getting all of these sorts of tests enabled with DB2:
https://github.com/hibernate/hibernate-reactive/issues/162